eng/pipelines/templates/scripts/create-tag-for-current-branch.ps1

```diff
@@ -0,0 +1,26 @@
+<#
+.SYNOPSIS
+.DESCRIPTION
+The script is to push current branch to remote tag. It is for backup purpose.
+.PARAMETER ReleaseTag
+The tag to back up current branch. The format is like mainbackup-2022-08-25
+.PARAMETER GithubUrl
+The github url for the working repo.
+#>
+[CmdletBinding(SupportsShouldProcess = $true)]
+param(
+  [Parameter(mandatory=$false)]
+  [string] $ReleaseTag = "mainbackup-$(Get-Date -Format 'yyyy-MM-dd')",
+  [Parameter(mandatory=$true)]
+  [string] $GithubUrl
+)
+
+# git push target branch to tag.
+Write-Host "git tag $ReleaseTag"
+git tag $ReleaseTag
+if ($PSCmdlet.ShouldProcess("git push origin $ReleaseTag", "Push local tag to remote tag.")) {
+    git push $GithubUrl $ReleaseTag | Write-Host
```

Comment:
Why do you need to pipe to Write-Host here?

Reply:
The output message is array instead of string. We can have the git output, or define the output ourselves. I would like to learn what the error message from git.

Reply:
By default everything that git outputs should be dumped out to the console so we shouldn't need to explicitly add this piping.

Reply:
Ah, I know what you mean here. My bad. 
I have the piece of codes originally in window agent inline powershell. They swallow my git message somehow. I will test in our 1ES agent pool to see if they can have git messages directly